core/frontend/src/main/scala/io/udash/properties/seq/SeqProperty.scala

```diff
@@ -99,6 +99,22 @@ trait ReadableSeqProperty[A, +ElemType <: ReadableProperty[A]] extends ReadableP
     new CombinedReadableSeqProperty(this, property, executionContext)
   }
 
+  /** Zips elements from `this` and provided `property` by combining every pair using provided `combiner`. */
+  def zip[B, O : ModelValue](property: ReadableSeqProperty[B, _ <: ReadableProperty[B]])(combiner: (A, B) => O): ReadableSeqProperty[O, ReadableProperty[O]] =
```

Comment:
I believe you could simplify the argument type to:
```scala 
ReadableSeqProperty[B, ReadableProperty[B]]
```
The second type parameter is covariant anyway.